lib/email-addresses.js

```diff
@@ -528,7 +528,21 @@ function parse5322(opts) {
             obsGroupList
         )());
     }
-
+    
+    function fromSpec() {
```

Comment:
// RFC 6854 2.1.  Replacement of RFC 5322, Section 3.6.2. Originator Fields
// from = "From:" (mailbox-list / address-list) CRLF